### ISSUE TITLE
shell: Add skip link

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#11569]
Cockpit should have a skip link, to immediately jump to the page's content.
https://webaim.org/techniques/skipnav/

**Comment by garrett** [comment]
Article about it: https://css-tricks.com/how-to-create-a-skip-to-content-link/

PF4 React way of doing this: https://www.patternfly.org/v4/documentation/react/components/skiptocontent

**Comment by KKoukiou** [comment]
@garrett I read the links - it's not clear to me where the skip link should be, in the start of the outer navigation?
Should it be visible or conditionally visible?

**Comment by garrett** [comment]
The skip link should be:
- "visible" for screen readers, always
- displayed when it has `:focus`, for tab keyboard support (else hidden)
- at the very top of the document as the first focusable element (well before navigation)

The question is if we target the iframe content directly or have two: one for jumping to the navigation, one for the iframe.

The purpose is to help keyboard users skip a lot of redundant content (on page loads), especially for screen reader usage (which is often, but not always, paired with keyboard navigation).

_Then_ comes the question of what happens to focus when someone activates a navigation link. Does it stay there? Does it go to the content? Does it jump back up to the top of the page? It's not exactly clear with SPAs (single page apps) or a hybrid approach (like Cockpit, where we're sort of a single page app and also a multi-page app).

**Comment by marusak** [comment]
@KKoukiou this would be either good to implement in https://github.com/cockpit-project/cockpit/pull/13663 or after it lands.
